src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -187,9 +189,16 @@ public static Properties getProperties() throws Exception {
     props.put(PRIVATE_KEY, privateKeyPem);
     props.put(ROLE, role);
     props.put(ACCOUNT_URL, getAccountURL());
+    props.put(BLOB_FORMAT_VERSION, getBlobFormatVersion());
     return props;
   }
 
+  private static int getBlobFormatVersion() {
+    return profile.has(BLOB_FORMAT_VERSION)
+        ? profile.get(BLOB_FORMAT_VERSION).asInt()
```

Comment:
Can we do a `getOrDefault` here?

Reply:
`profile` is a json node, I do not see it having `getOrDefault`